content/pages/qbot.md

```diff
@@ -0,0 +1,55 @@
+Title: Using Q Bot with the-asf Slack channels
+
+The Infrastructure team's **Q Bot** is a Slack assistant that can simplify both important and frivolous tasks on Slack channels in the `the-asf` workspace. It can support requests in channels for PMCs, ASF committees, the ASF Board and `asfmembers`.
+
+The name references the tool-maker who provides weapons and gadgets for Agent 007 in the James Bond movies.
+
+## What Q Bot can do
+Q Bot has an evolving set of functions:
+  - Providing to the channel notifications related to your project's Jira tickets - new ticket, new comment, ticket resolved.
+  - Starting a huddle in a channel workspace
+  - Adding people to private channels
+
+It also has a set of 'fun' features, such as rolling dice or flipping a coin.
+
+## Setting up Q Bot in your PMC's channels
+If you want to have Q Bot available in your PMC's channel, create a Jira ticket for Infra with the request. Include this information:
+  - The name of the channel or channels where Q Bot notifications about Jira tickets related to your projects should appear
+  - How you want notifications to appear. There are five options:
+      - No notifications
+      - All notifications
+      - `nodescription`: All notifications without the title of the ticket, but not its description
```

Comment:
**with** the title of the ticket, but not... (I think)

Reply:
Fixed